packages/amplify-category-api/src/commands/api.ts

```diff
@@ -55,18 +55,6 @@ export const run = async (context: $TSContext) => {
     {
       name: 'override',
       description: 'Generates overrides file to apply custom modifications to CloudFormation',
-    },
-    {
-      name: 'import',
-      description: 'Imports existing datasource to GraphQL API',
-    },
-    {
-      name: 'generate-schema',
-      description: 'Generates the GraphQL schema from the Data Source',
-    },
-    {
-      name: 'update-secrets',
-      description: 'Updates the API plugin related secrets',
```

Comment:
Can we comment these lines with note so it is easy to enable when required?

Reply:
We probably need these reviewed before release since I see some scope for improvement here. Can we re-visit them when these are ready?